In [ ]:
import pandas as pd
df = pd.read_excel('Entregable 2 - Documentación extra.xlsx')

In [2]:
df.head()

,",country,description,designation,points,price,province,region_1,region_2,taster_name,taster_twitter_handle,title,variety,winery",Unnamed: 1,Unnamed: 2,Unnamed: 3,Unnamed: 4
0,"0,Italy,""Aromas include tropical fruit, broom,...",NaN,NaN,NaN,NaN
1,"1,Portugal,""This is ripe and fruity, a wine th...",NaN,NaN,NaN,NaN
2,"2,US,""Tart and snappy, the flavors of lime fle...",NaN,NaN,NaN,NaN
3,"3,US,""Pineapple rind, lemon pith and orange bl...",NaN,NaN,NaN,NaN
4,"4,US,""Much like the regular bottling from 2012...",NaN,NaN,NaN,NaN


Aparentemente el archivo es un excel pero mal configurado. Parecería ser que todo el contenido está en una sola columna con sus valores separados por coma. Voy a revisar si las otras 3 columnas (unnamed) tienen algun dato.

In [5]:
df.info()

<class 'pandas.DataFrame'>
RangeIndex: 129975 entries, 0 to 129974
Data columns (total 5 columns):
 #   Column                                                                                                                           Non-Null Count   Dtype
---  ------                                                                                                                           --------------   -----
 0   ,country,description,designation,points,price,province,region_1,region_2,taster_name,taster_twitter_handle,title,variety,winery  129975 non-null  str  
 1   Unnamed: 1                                                                                                                       2956 non-null    str  
 2   Unnamed: 2                                                                                                                       159 non-null     str  
 3   Unnamed: 3                                                                                                     

Efectivamente, todos los Unnamed tienen datos. Y esa cantidad de datos o filas no nulas parece ir en cascada, es decir del Unnamed 1 al 5 la cantidad de filas no nulas va decreciendo, siendo 2956 en el 1 y 1 en el 5

In [20]:
mascara = df['Unnamed: 1'].notna()
primer_notna = df.loc[mascara].iloc[0]

In [21]:
for x in primer_notna:
    print(x)

48,US,"This bottling resembles the New Zealand paradigm of Sauvignon Blanc, bearing aromas of grapefruit, passion fruit and kiwi
 a sprinkling of graham cracker adds interest. The wine hits the palate like a fleshy fist, with an intense, grassy gooseberry flavor that provides plenty of punch. Pair with apricot-glazed roasted chicken.",,86,16.0,Virginia,Monticello,,,,Trump 2011 Sauvignon Blanc (Monticello),Sauvignon Blanc,Trump
nan
nan
nan


Parece que el motivo de las columnas unnamed es que el texto se parte en dos por una coma mal parseada como delimitador, partiendo asi el texto. 